modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusStateUpdater.scala

```diff
@@ -58,52 +58,67 @@ object ConsensusStateUpdater {
 
     private val logger = Slf4jLogger.getLoggerFromClass(ConsensusStateUpdater.getClass)
 
+    /** Pair of (maybeUpdatedState, effect) */
```

Comment:
eventually alias:

```scala
type MaybeUpdatedState = Option[ConsensusState[Key, Artifact]]
```

it is used in many places so putting comment here isn't helpful as we need to keep scrolling to that comment. I do not have any better idea ;<

Reply:
You're right. It would be better to use the alias you've suggested since it's used more (I would just rename it to `MaybeState` since it's also used also before the update, not just after the update). Then we could remove the `MaybeStateResult` and just keep it as `(MaybeState, F[Unit])` since it's not used as much.

Reply:
I will add this in another PR.